web/src/main/webapp/ui/menu.ts

```diff
@@ -130,6 +160,56 @@ abstract class BaseMenu<MI extends BaseMenuItem> implements IHtmlElement {
         return cell;
     }
 
+    /**
+     * Inserting subMenu must be done in the end of all parent menu insert. If the
+     * submenu exceed the length of all main menu, we need to add dummyMenu
```

Comment:
I don't understand the role of dummyMenu, can you explain it?

Reply:
I put the explanation here: https://github.com/vmware/hillview/pull/650#discussion_r462205270